lib/modules/hide_downvote.js

```diff
@@ -0,0 +1,13 @@
+HNSpecial.settings.registerModule("hide_downvote", function () {
+  function hideDownvoteButton () {
+    var arrows = _.toArray(document.getElementsByClassName('votearrow'));
+
+    arrows.forEach(function(arrow) {
+      if (arrow.classList.contains('rotate180')) {
+        arrow.parentElement.remove();
+      }
+    });
+  }
+
+  hideDownvoteButton();
```

Comment:
I had initially been looking at another module which was subscribing to events, and I followed that design in case there had been a "new comments" event or similar. I can remove the extraneous definition, though.